leakcanary-android-release/api/leakcanary-android-release.api

```diff
@@ -4,8 +4,6 @@ public final class com/squareup/leakcanary/release/BuildConfig {
 	public static final field GIT_SHA Ljava/lang/String;
 	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
 	public static final field LIBRARY_VERSION Ljava/lang/String;
-	public static final field VERSION_CODE I
```

Comment:
Why not disable`BuildConfig` generating in unnecessary modules ?

Reply:
No reason, sounds like we should?

Reply:
BuildConfig is not useful for users.

Reply:
I know. I was saying "there was particular reason why we didn't disable it, except we just didn't think of doing that, and probably it's indeed worth disabling".

Reply:
ok